actionpack/test/controller/renderer_test.rb

```diff
@@ -96,13 +95,20 @@ class RendererTest < ActiveSupport::TestCase
   end
 
   test "rendering with user specified defaults" do
-    ApplicationController.renderer.defaults.merge!(hello: "hello", https: true)
+    ApplicationController.renderer.defaults.merge!(hello: "hello", url_scheme: 'https')
     renderer = ApplicationController.renderer.new
     content = renderer.render inline: "<%= request.ssl? %>"
 
     assert_equal "true", content
   end
 
+  test "return valid asset url with defaults" do
+    renderer = ApplicationController.renderer
+    content  = renderer.render template: "test/render_file_with_asset"
+
+    assert_equal "http://example.org/asset.jpg", content
```

Comment:
Can you also test the scenario where a renderer is initialized with `url_scheme: "https"`?